applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -205,6 +207,15 @@ def PrepareModelPart(self):
                                                   self.settings["nodal_smoothing"].GetBool())
 
         self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.STEP, 0)
+        self.computing_model_part_name = "porous_computational_model_part"
+
+        # Create list of sub sub model parts (it is a copy of the standard lists with a different name)
```

Comment:
The comment had me wondering, but it does not really add information. Removing will prevent that it becomes wrong comment.

Reply:
Removed